cves/kernel/CVE-2013-2635.yml

```diff
@@ -1,393 +1,101 @@
   announced:
   url:
 reviews: []
-bugs_instructions: |
-  What bugs are involved in this vulnerability?
-
-  Please list bug IDs to https://bugzilla.kernel.org/
-
-  Bug ID's can appear in several places:
-    * Mentioned in commit messages
-    * Mentioned in mailing list discussions
-    * References from NVD entry
-    * Various other places
-bugs: []
-fixes_instructions: |
-  Please put the commit hash in "commit" below.
-
-  This must be a git commit hash from the systemd source repo, a  40-character
-  hexademical string/
-
-  Place any notes you would like to make in the notes field.
+bugs: ["https://bugzilla.redhat.com/show_bug.cgi?id=924690"]
 fixes:
-- commit:
-  note:
-- commit:
-  note:
 - commit: 84d73cd3fb142bf1298a8c13fd4ca50fd2432372
-  note: |
-    Taken from NVD references list with Git commit. If you are
-    curating, please fact-check that this commit fixes the vulnerability and replace this comment with 'Manually confirmed'
-vcc_instructions: |
-  The vulnerability-contributing commits.
-
-  These are found by our tools by traversing the Git Blame history, where we
-  determine which commit(s) introduced the functionality.
-
-  Look up these VCC commits and verify that they are not simple refactorings,
-  and that they are, in fact introducing the vulnerability into the system.
-  Often, introducing the file or function is where the VCC is, but VCCs can be
-  anything.
-
-  Place any notes you would like to make in the notes field.
+  note: Manually confirmed
 vccs:
 - commit: 5f8444a3fa617076f8da51a3e8ecce01a5d7f738
   note: Discovered automatically by archeogit.
 - commit: ebc08a6f47ee76ecad8e9f26c26e6ec9b46ca659
   note: Discovered automatically by archeogit.
-upvotes_instructions: |
-  For the first round, ignore this upvotes number.
-
-  For the second round of reviewing, you will be giving a certain amount of
-  upvotes to each vulnerability you see. Your peers will tell you how
-  interesting they think this vulnerability is, and you'll add that to the
-  upvotes score on your branch.
 upvotes:
 unit_tested:
-  question: |
-    Were automated unit tests involved in this vulnerability?
-    Was the original code unit tested, or not unit tested? Did the fix involve
-    improving the automated tests?
-
-    For code: and fix: - your answer should be boolean.
-
-    For the code_answer below, look not only at the fix but the surrounding
-    code near the fix in related directories and determine if and was there were
-    unit tests involved for this subsystem.
-
-    For the fix_answer below, check if the fix for the vulnerability involves
-    adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: false
+  fix: false
+  fix_answer: |
+    I wasn't able to find unit tests for this module, and it doesn't look like tests were involved in finding this bug. (It appeared to be found by 
+    the use of external applications)
```

Comment:
I would avoid first person and parantheses. You could just say it was found by the use of external applications. Saying "It appeared" does not instill much confidence in your answer